include/roboteam_world/WorldHandler.h

```diff
@@ -16,8 +16,8 @@ class WorldHandler {
   proto::Publisher<proto::SSL_Referee> *ref_pub;
   proto::Publisher<proto::SSL_GeometryData> *geom_pub;
 
-  WorldBase *world;
-  KalmanFilter *KF;
+  double lastPacketTime;
+  WorldFilter *KF;
```

Comment:
Initialize to nullptr to prevent UB. 
```suggestion
  WorldFilter *KF = nullptr;
```

Reply:
nice catch!